bootstrapvz/common/tasks/grub.py

```diff
@@ -27,9 +27,9 @@ def run(cls, info):
 		grub_def = os.path.join(info.root, 'etc/default/grub')
 		sed_i(grub_def, '^#GRUB_TERMINAL=console', 'GRUB_TERMINAL=console')
 		sed_i(grub_def, '^GRUB_CMDLINE_LINUX_DEFAULT="quiet"',
-		                'GRUB_CMDLINE_LINUX_DEFAULT="console=hvc0"')
+		                'GRUB_CMDLINE_LINUX_DEFAULT="init=/bin/systemd console=hvc0 console=ttyS0"')
```

Comment:
We should probably start commenting what all this does/effect it has :-)

wouldn't the second `console=` overwrite the first one?